src/realm/db.cpp

```diff
@@ -229,11 +232,12 @@ struct VersionList {
                 continue;
             if (rc->count_frozen == 0 && rc->count_live == 0) {
                 // entry is becoming unreachable.
-                // if it is also yonger than a reachable version, so set 'any_new_unreachables' to trigger backdating
+                // if it is also yonger than a reachable version, then set 'any_new_unreachables' to trigger
```

Comment:
```suggestion
                // if it is also younger than a reachable version, then set 'any_new_unreachables' to trigger
```

I think you missed this in my earlier comment